portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,24 @@
+// package com.google.sps.servlets;
+
```

Comment:
File is completely commented. Should you remove the file?

Reply:
Yes, so sorry I missed that. Will remove the file asap.

Reply:
Deleted the concerned file, kindly review.